runtime-testsuite/resources/org/antlr/v4/test/runtime/helpers/Test.ts.stg

```diff
@@ -0,0 +1,53 @@
+import { default as antlr4 } from 'antlr4';
+import <lexerName> from './<lexerName>.js';
+<if(parserName)>
+import <parserName> from './<parserName>.js';
+<if(useListener)>
+import <grammarName>Listener from './<grammarName>Listener.js';
+<endif>
+<if(useVisitor)>
+import <grammarName>Visitor from './<grammarName>Visitor.js';
+<endif>
```

Comment:
Why generated files are written in JavaScript instead of TypeScript? It looks weird.

Reply:
Agreed it looks weird, but that's the way it works